.github/workflows/node-zxf-deploy-integration.yaml

```
@@ -40,7 +49,7 @@ jobs:
         uses: fjogeleit/http-request-action@bf78da14118941f7e940279dd58f67e863cbeff6 # v1.16.3
         with:
           url: ${{ secrets.RELEASE_JENKINS_INTEGRATION_URL  }}
-          data: ${{ inputs.event }}
+          data: ${{ inputs }}
```

this may need to be something like follows:
```suggestion
          data: ${{ toJSON(inputs) }}
```

---

Updated.